ctf/packages/web/.env.local.example

```
@@ -54,13 +54,25 @@ RAILWAY_STAGING_AUTH_SIGN_IN_URL=
 RAILWAY_PROD_NEXT_PUBLIC_AUTH_PUBLISHABLE_KEY=
 RAILWAY_PROD_AUTH_SECRET_KEY=
 RAILWAY_PROD_AUTH_SIGN_IN_URL=
+
+# VERCEL (Vercel staging frontend)
+VERCEL_NEXT_PUBLIC_AUTH_PUBLISHABLE_KEY=
+VERCEL_AUTH_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VERCEL_AUTH_SECRET_KEY key should go before the VERCEL_NEXT_PUBLIC_AUTH_PUBLISHABLE_KEY key

(UnorderedKey)